recipes/sdds/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is already present in the PyPI tarball, so it shouldn't be included here.